firexapp/firex_subprocess.py

```diff
@@ -492,6 +511,13 @@ def _collect_stats():
 
                 # Inactivity timeout check
                 current_output_size = os.fstat(f.fileno()).st_size
+                for file in found_monitor_activity_files:
+                    try:
+                        with open(file, 'r') as fp:
+                            current_output_size += os.fstat(fp.fileno()).st_size
```

Comment:
Is there a benefit to using `fstat` over `os.stat(file).st_size` here? I think we use `fstat` above because we already have `f` open.